scripts/i18n/PROGRESS.template.md

```diff
@@ -0,0 +1,23 @@
+## For New Translators
+
+To translate a page:
+
+1. Check that no one else has claimed your page in the checklist and comments below.
+2. Comment below with the name of the page you would like to translate. **Please take only one page at a time**.
+3. Clone this repo, translate your page, and submit a pull request!
```

Comment:
I almost linked this to the Gatsby contributing docs, but I realized it would be talking about the wrong repo. Perhaps a future enhancement could add a section on translation repos: https://www.gatsbyjs.org/contributing/how-to-open-a-pull-request/